src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.CodeModification/CodeModificationStep.cs

```diff
@@ -96,13 +96,28 @@ private void EditCodeModifierConfig(CodeModifierConfig codeModifierConfig)
             return;
         }
 
+        //modify CodeSnippets 'CheckBlock', 'Parent', and 'Block'
```

Comment:
This whole block of code feels like something that should be encapsulated somewhere - perhaps on the `CodeModifierConfig` class itself, or on an extension method for it, or something. Just seems like a lot of detail that `CodeModificationStep` shouldn't need to know about beyond needing to call `codeModifierConfig.PerformReplacements(CodeModifierProperties)` or something like that. Would probably make it a little more reasonable to write tests for it too, since it'd be public methods on the impacted class rather than private methods in a ScaffoldStep.

Reply:
ok yes will move this on the class `CodeModifierConfig` itself